aten/src/ATen/DLConvertor.cpp

```diff
@@ -154,6 +154,8 @@ struct ATenDLMTensor {
 };
 
 void deleter(DLManagedTensor* arg) {
+  arg->manager_ctx = nullptr;
```

Comment:
if you set `arg->manager_ctx` to `nullptr`, `delete static_cast<ATenDLMTensor*>(arg->manager_ctx)` won't work as before?